opentreemap/treemap/models.py

```diff
@@ -1230,6 +1230,7 @@ class Boundary(models.Model):
                                       db_index=True)
 
     canopy_percent = models.FloatField(null=True)
+    searchable = models.BooleanField(default=True)
```

Comment:
Sorry. Backwards compatible from a blue/green standpoint. During a deploy we run the migrations and test the "dark" stack. During that time, the old code is running with the new schema and inserts would fail.

It is a little academic in this situation, because there is no user-facing way to insert into the boundaries table. I have been bitten by migration problems during deploys enough times that I like to err on the side of safety and keep all new fields nullable when they are first introduced.